example-apps/app-with-external-docker-images/local-service/service.yml

```diff
@@ -12,3 +12,7 @@ restart:
 messages:
   sends:
   receives:
+
+development:
+  scripts:
+    run: node server.js
```

Comment:
Didn't we say this line should be in the Dockerfile?

Reply:
It's in the dockerfile for production, but since the dev dockerfile may do many things (test, lint, run specs, run, etc...) we must specify them here.